docker/test/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '2.1'
+
+networks:
+  iam-net-test:
+    driver: bridge
+    name: "iam-net-test"
+
+services:
+  iam-server-test:
+    build:
+      context: ../../
+      dockerfile: docker/development/Dockerfile
+    container_name: 'iam-server-test'
+    environment:
+      GRAPH_DB_HOST: graphdb-test
+      GRAPH_DB_CONNECTION_TIMEOUT: 30000
```

Comment:
I think you should just give it a timeout of 15s. 30s is too long 😫

Reply:
Changing.